libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts

```diff
@@ -72,10 +75,12 @@ const abandonSeedAddresses: Record<string, string> = {
  * DO NOT USE AS RECIPIENT OR SIGN TRANSACTIONS INTO THEM
  * @param {*} currencyId
  */
+// TODO: signature should be  getAbandonSeedAddress(currencyId: CryptoCurrencyId)
```

Comment:
Same question, what's preventing this currently?

Reply:
A lot of impact outside of AVAX, I started but then reverted at the sight of files changed